src/Components/Play/HumanVsHuman.jsx

```diff
@@ -536,30 +572,68 @@ class HumanVsHuman extends Component {
     }
 
     // Deals with the opponent forfeiting the game
-    forfeitReceived = () => {
-        // Show opponent forfeit window
-        const confBox = document.getElementById("opponent-forfeit-confirmation");
+    forfeitReceived = (forfeit_user_id) => {
+        const userid = this.props.userid;
+        const gameid = this.props.gameid;
 
-        // Open the confirmation window
-        confBox.style.display = "block";
+        if (userid !== forfeit_user_id) {
+            axios.get(`https://negativei2-server.herokuapp.com/getgame/${gameid}`)
```

Comment:
Should use ``server.get(`/getgame/${gameid}`)`` here instead of `axios`. _See #110._